docs/content/intro/tutorial-preview-environments.md

```diff
@@ -0,0 +1,433 @@
+---
+title: "Tutorial: Preview Environments"
+description: Automate Creation of Development Preview Environments
+section: intro
+---
+
+In this tutorial we'll create a workflow for creating Development Preview Environments.
+
+Preview Environment is a dedicated Kubernetes Namespace where all of your applications 
+and their dependencies are deployed. We will create a pipeline for both creation of these ephemeral environments 
+as well as auto-deployment of the applications themselves.
+
+Preview Environments are a great place for early experimentation where changes to your 
+application under development can be tested against production version of applications you're depending on.
+
+## Setup
+
+For the purpose of this tutorial we're assuming you have a Kubernetes cluster up and running and Brigade components are deployed to `brigade` namespace within it. We'll be relying on GitHub webhooks so make sure that `brigade-github-app` is also installed.  
+
+[Docker for Desktop's Kubernetes](https://docs.docker.com/docker-for-mac/kubernetes/) 
+cluster is sufficient to perform all steps from this tutorial. To accept incoming GitHub Webhooks to your desktop cluster you can use free version of [Ngrok](https://ngrok.com/) service to establish secure tunnelling. Follow this [excellent guide](https://stefanprodan.com/2018/expose-kubernetes-services-over-http-with-ngrok/) to set this up.
+
+Brigade's [brig](../../install/#brig) cli utility should be present on your machine.
+
+## Git Repositories
+
+We will use two GitHub repositories:
+
+- [brigade-tutorial-config](https://github.com/brigadecore/brigade-tutorial-config): containing orchestration responsible for managing brigade projects and creating new environments.
+- [brigade-tutorial-app](https://github.com/brigadecore/brigade-tutorial-app): example microservice used to demonstrate release process.
+
+`GITHUB_SHARED_SECRET` and `GITHUB_TOKEN` environment variables will be used throughout this tutorial and you'll have to ensure they are set correctly. To generate GitHub token follow [this article](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line).
+
+## Config Brigade Project
+
+Let's create new Brigade Project for our `brigade-tutorial-config` repository.
+
+```sh
+# Ensure brigade charts are added to your helm registry:
+$ helm repo add brigade https://brigadecore.github.io/charts \
+  --kube-context=docker-for-desktop
+
+# Create new Brigade Project by installing brigade-project chart:
+$ helm upgrade brigade-config brigade/brigade-project --install \
```

Comment:
Could you also add instructions on how to use `brig project create`?

Not necessarily required, but it might be helpful.

Reply:
I've changed it to use `brig project create` only.